src/views/Auth/Login.jsx

```diff
@@ -16,8 +16,20 @@ export default function Login() {
   const { from } = location.state || { from: { pathname: '/' } };
 
   const handleLogin = (event) => {
-    event.preventDefault();
-    const loginWasSuccessful = auth.login(formState.email, formState.password);
+    try {
+      event.preventDefault();
+      const loginWasSuccessful = auth.login(
+        formState.email,
+        formState.password
+      );
+      if (loginWasSuccessful) {
+        history.replace('/');
```

Comment:
Consider adding the from variable in the history.replace method to direct the user where they originally wanted to go.